docs/ADR-52-content-new-sync.md

```diff
@@ -0,0 +1,107 @@
+const allServers = await  getAllCatalystFromDao() // gets that info from the contract
+
+while(true) {
+  for (server of allServers) {
+    // GET /deployments : first time will be all, then only diff
+    allDeployments = await server.getNewDeployments()
+    deployLocally(allDeployments)
+    server.updateLastSeenTimestamp()
+  }
+}
+```
+
+This behaviour is correct as no info is missed and we can ensure that every new entity deployed will be discoverable. A characteristic from this solution is that not only current active entities will be synchronized but also all the history, this can be beneficial if you want to store all the historical data from Decentraland but could be a disadvantage if you don’t care or you need to use less space.
+
+#### Stability issues
+Using that way to synchronize we found out that when increasing the amount of users, this implied an increase of the amount of deployments (profiles mostly) and the servers weren’t responding well to that: the servers that need to retrieve the deployments experienced a huge amount of usage of CPU and also the queries to the DB to get all the deployments  and as all servers sync with each other that implied a maximum amount of users of 100/140 users per catalyst which was  below of what we expected and we needed.
+
+#### Time to be up
+When there were few deployments all of this worked fine, but when we reached 2M deployments and 50k deployments daily, the time spended retrieving all the history from the other servers and deploying that locally was always bigger than the new amount of deployments. So if you wanted to create a new Catalyst and sync it from scratch it could take you a month and you will never be synced.
+
+
+## Decision
+
+### New Sync
+To build the new sync we put the focus on making the Catalyst work, so we left behind the concept of having all the historical deployments everywhere. This doesn’t mean that we lost that data, it is backed up in s3. But to implement the sync we only took into account the deployed entities that are active.
+
+For the new sync we continue communicating all the catalysts between them, but in a more efficient way:
+1. We decreased the amount of requests for bootstrapping from 400 to 1.
+2. Deprecate the endpoint /deployments (which was the most expensive in the db)
+
+```
+// Synchronization code
+const allServers = await  getAllCatalystFromDao() // gets that info from the contract
+
+// Bootstrapping
+for (server of allServers) {
+  // GET /snapshots : get all entities that are active
+  allDeployments = await server.getSnapshots() 
+  deployLocally(allDeployments)
+  server.updateLastSeenTimestamp() // the biggest timestamp seen in snapshots
+}
+
+while(true) {
+  for (server of allServers) {
+    // GET /pointer-changes : always only the new deployments
+    allDeployments = await server.getNewDeployments() 
+    deployLocally(allDeployments)
+    server.updateLastSeenTimestamp() // the biggest timestamp seen in pointer-changes
+  }
+}
```

Comment:
I think this explains better the concurrency of the synchronization against servers

```suggestion

async function sync(remoteServer) {
  // Bootstrapping
  
  // GET /snapshots : get all entities that are active
  allDeployments = await remoteServer.getSnapshots() 
  deployLocally(allDeployments)
  remoteServer.updateLastSeenTimestamp() // the biggest timestamp seen in snapshots

  // Syncing
  while(remoteServer.isOnline()) {
    // GET /pointer-changes : always only the new deployments
    allDeployments = await remoteServer.getNewDeployments() 
    await deployLocally(allDeployments)
    remoteServer.updateLastSeenTimestamp() // the biggest timestamp seen in pointer-changes
  }
}

for (server of allServers) {
  // concurrently
  sync(server)
}
```